skiller.sh

```diff
@@ -135,7 +135,49 @@ def create_skill():
     return $class_name()
 """ > "$skill_dir/__init__.py"
 
-echo -e "*.pyc\nsettings.json\n" > $skill_dir/.gitignore
+echo '{
+	"name": "'${capital_desc}'",
+	"skillMetadata": {
+		"sections": [
+		{
+		 "name": "Section Friendly Display Name for Home.Mycroft.AI",
+		 "fields": [
+					{
+						"name": "internal_python_variable_name",
+						"type": "text",
+						"label": "Setting Friendly Display Name",
+						"value": "",
+						"placeholder": "demo prompt in the input box"
+					}
+				]
+			},
+			{
+				"name": "Login --Another Section But This One Has Two Values--",
+				"fields": [
+					{
+			"type": "label",
```

Comment:
One last thing, some of the json looks like it got formatted funny. ([here's a corrected version](https://hastebin.com/romofevuji.json)). Once that's fixed, it looks good to go :+1: